.github/workflows/tests.yaml

```diff
@@ -28,34 +28,11 @@ jobs:
 
       # TODO: Cache yarn install
 
-      - name: Set variables
-        id: variables
-        working-directory: packages/${{ matrix.package }}
-        run: |
-          echo "DM_CORE_VERSION=$(cat package.json | jq '.["dependencies"]["@development-framework/dm-core"]')" >> "$GITHUB_OUTPUT"
-          echo "TEST_COMMAND=$(cat package.json | jq '.["scripts"]["test"]')" >> "$GITHUB_OUTPUT"
-
       - name: 'Install dependencies'
-        working-directory: packages/${{ matrix.package }}
         run: yarn install
 
-      #      - name: 'Copy local variant of dm-core to node_modules'
-      #        working-directory: packages/${{ matrix.package }}
-      #        if: steps.variables.outputs.DM_CORE_VERSION != 'null'
-      #        run: |
-      #          cd ../dm-core
-      #          yarn install --no-lockfile
-      #          yarn build
-      #          cd ../dm-core-plugins
-      #          rm -r node_modules/@development-framework/dm-core/*
-      #          cp -r ../dm-core/dist node_modules/@development-framework/dm-core/dist
-      #          cp ../dm-core/package.json node_modules/@development-framework/dm-core/package.json
-
-      - name: 'yarn tsc'
```

Comment:
I guess this should still be here, to test that the typescript works?

Reply:
Build in the individual packages is `yarn tsc`, wouldn't that be the same? 🙂